### PR TITLE
feat: add Closed proposals section

### PR DESCRIPTION
**Pull request by Sekhmet**
Closes: https://github.com/snapshot-labs/sx-ui/issues/325

## Screenshots

image

### CODE REVIEW

File: src/components/ProposalsList.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  route?: {
+    name: string;
+    linkTitle: string;
+  };
+}>();
+
+const limit = computed(() => {
+  if (props.limit === false) return Infinity;
+
+  return props.limit || 3;
```
Comment (bonustrack) [inline src/components/ProposalsList.vue:19]: It seem that the limit of 3 doesnt work for me, I can see 4 closed proposals on this space:
https://bafybeibh4naq5yi67sa43jmlpoojvh646f5oyrsc6fgqd7h6lcpy7i4gd4.on.fleek.co/#/0x04b7cff71219e275676e0ca23579f41b99dd1d1bd01adc7d7f1bc917d448e57d

**Review by bonustrack — COMMENTED** [review]
This works but we may not want to rely on the UI to filter the proposals for too long, it will be more tricky when there is more proposals than what we query in a single request. We could query the active and closed proposals from the API, it will probably break Pinia, we wont be able to reuse the same data from the overview page in the proposals page, not sure there is solution for this.

File: src/views/Space/Overview.vue [hunk 2]
```diff
@@ -1,19 +1,37 @@
 <script setup lang="ts">
 import { onMounted, computed } from 'vue';
 import { useProposalsStore } from '@/stores/proposals';
-import { Space } from '@/types';
+import { Space, Proposal as ProposalType } from '@/types';
+
+const PROPOSALS_LIMIT = 4;
```
Comment (Sekhmet) [inline src/views/Space/Overview.vue:6]: We need to fetch one more than we want to display to be able to detect if we need See more link.

**Review by bonustrack — APPROVED** [review]
Works well

**Pull request merged**